openmc/data/data.py

```diff
@@ -3,89 +3,89 @@
+    'O18': 0.0020004, 'F19': 1.0, 'Ne20': 0.9048,
     'Ne21': 0.0027, 'Ne22': 0.0925, 'Na23': 1.0,
-    'Mg24': 0.7899, 'Mg25': 0.1, 'Mg26': 0.1101,
-    'Al27': 1.0, 'Si28': 0.92223, 'Si29': 0.04685,
-    'Si30': 0.03092, 'P31': 1.0, 'S32': 0.9499,
-    'S33': 0.0075, 'S34': 0.0425, 'S36': 0.0001,
-    'Cl35': 0.7576, 'Cl37': 0.2424, 'Ar36': 0.003336,
+    'Mg24': 0.78951, 'Mg25': 0.1002, 'Mg26': 0.11029,
+    'Al27': 1.0, 'Si28': 0.9222968, 'Si29': 0.0468316,
+    'Si30': 0.0308716, 'P31': 1.0, 'S32': 0.9504074,
+    'S33': 0.0074869, 'S34': 0.0419599, 'S36': 0.0001458,
+    'Cl35': 0.757647, 'Cl37': 0.242353, 'Ar36': 0.003336,
     'Ar38': 0.000629, 'Ar40': 0.996035, 'K39': 0.932581,
     'K40': 0.000117, 'K41': 0.067302, 'Ca40': 0.96941,
     'Ca42': 0.00647, 'Ca43': 0.00135, 'Ca44': 0.02086,
-    'Ca46': 4e-05, 'Ca48': 0.00187, 'Sc45': 1.0,
+    'Ca46': 0.00004, 'Ca48': 0.00187, 'Sc45': 1.0,
     'Ti46': 0.0825, 'Ti47': 0.0744, 'Ti48': 0.7372,
     'Ti49': 0.0541, 'Ti50': 0.0518, 'V50': 0.0025,
     'V51': 0.9975, 'Cr50': 0.04345, 'Cr52': 0.83789,
     'Cr53': 0.09501, 'Cr54': 0.02365, 'Mn55': 1.0,
     'Fe54': 0.05845, 'Fe56': 0.91754, 'Fe57': 0.02119,
-    'Fe58': 0.00282, 'Co59': 1.0, 'Ni58': 0.68077,
-    'Ni60': 0.26223, 'Ni61': 0.011399, 'Ni62': 0.036346,
-    'Ni64': 0.009255, 'Cu63': 0.6915, 'Cu65': 0.3085,
+    'Fe58': 0.00282, 'Co59': 1.0, 'Ni58': 0.680769,
+    'Ni60': 0.262231, 'Ni61': 0.011399, 'Ni62': 0.036345,
+    'Ni64': 0.009256, 'Cu63': 0.6915, 'Cu65': 0.3085,
     'Zn64': 0.4917, 'Zn66': 0.2773, 'Zn67': 0.0404,
     'Zn68': 0.1845, 'Zn70': 0.0061, 'Ga69': 0.60108,
-    'Ga71': 0.39892, 'Ge70': 0.2057, 'Ge72': 0.2745,
-    'Ge73': 0.0775, 'Ge74': 0.365, 'Ge76': 0.0773,
-    'As75': 1.0, 'Se74': 0.0089, 'Se76': 0.0937,
-    'Se77': 0.0763, 'Se78': 0.2377, 'Se80': 0.4961,
-    'Se82': 0.0873, 'Br79': 0.5069, 'Br81': 0.4931,
+    'Ga71': 0.39892, 'Ge70': 0.2052, 'Ge72': 0.2745,
+    'Ge73': 0.0776, 'Ge74': 0.3652, 'Ge76': 0.0775,
+    'As75': 1.0, 'Se74': 0.0086, 'Se76': 0.0923,
+    'Se77': 0.076, 'Se78': 0.2369, 'Se80': 0.498,
+    'Se82': 0.0882, 'Br79': 0.50686, 'Br81': 0.49314,
     'Kr78': 0.00355, 'Kr80': 0.02286, 'Kr82': 0.11593,
     'Kr83': 0.115, 'Kr84': 0.56987, 'Kr86': 0.17279,
     'Rb85': 0.7217, 'Rb87': 0.2783, 'Sr84': 0.0056,
     'Sr86': 0.0986, 'Sr87': 0.07, 'Sr88': 0.8258,
     'Y89': 1.0, 'Zr90': 0.5145, 'Zr91': 0.1122,
     'Zr92': 0.1715, 'Zr94': 0.1738, 'Zr96': 0.028,
-    'Nb93': 1.0, 'Mo92': 0.1453, 'Mo94': 0.0915,
-    'Mo95': 0.1584, 'Mo96': 0.1667, 'Mo97': 0.096,
-    'Mo98': 0.2439, 'Mo100': 0.0982, 'Ru96': 0.0554,
+    'Nb93': 1.0, 'Mo92': 0.14649, 'Mo94': 0.09187,
+    'Mo95': 0.15873, 'Mo96': 0.16673, 'Mo97': 0.09582,
+    'Mo98': 0.24292, 'Mo100': 0.09744, 'Ru96': 0.0554,
     'Ru98': 0.0187, 'Ru99': 0.1276, 'Ru100': 0.126,
     'Ru101': 0.1706, 'Ru102': 0.3155, 'Ru104': 0.1862,
-    'Rh103': 1.0, 'Pd102': 0.0102, 'Pd104': 0.1114,
+    'Rh103': 1, 'Pd102': 0.0102, 'Pd104': 0.1114,
```

Comment:
It might be a good idea to leave the abundance of Rh103 as a float `1.0` rather than an integer `1`. The code for `Material.add_nuclide(...)` should be able to accept an integral atom percent, but I think it is safer to use a float in case a user perform some math with the atom percents within a material, and assumes that each is a float (as I have done in the past).